.gitlab-ci.yml

```diff
@@ -480,60 +333,111 @@ build/cuda102/nompi/intel/cuda/debug/static:
     BUILD_SHARED_LIBS: "OFF"
     CUDA_ARCH: 35
 
-# cuda 11.0 and friends
+# cuda 11.0 and friends on HoreKa with tests
 build/cuda110/mvapich2/gcc/cuda/debug/shared:
-  <<: *default_build_with_test
   extends:
+    - .build_template
+    - .default_variables
     - .full_test_condition
     - .use_gko-cuda110-mvapich2-gnu9-llvm9-intel2020
   variables:
-    <<: *default_variables
     BUILD_OMP: "ON"
     BUILD_CUDA: "ON"
     BUILD_MPI: "ON"
     BUILD_TYPE: "Debug"
     FAST_TESTS: "ON"
-    CUDA_ARCH: 61
+    CUDA_ARCH: 80
+    USE_NAME: "cuda110-mvapich2-gcc-${CI_PIPELINE_ID}"
+    KEEP_CONTAINER: "ON"
+    USE_SLURM: 0
+
+test/cuda110/mvapich2/gcc/cuda/debug/shared:
+  extends:
+    - .horeka_test_template
+    - .default_variables
+    - .full_test_condition
+    - .use_gko-cuda110-mvapich2-gnu9-llvm9-intel2020
+  variables:
+    USE_NAME: "cuda110-mvapich2-gcc-${CI_PIPELINE_ID}"
+    SLURM_PARTITION: "accelerated"
+    SLURM_GRES: "gpu:1"
```

Comment:
Could we already run multi-GPU MPI CI by increasing the number of GPUs here ?

Reply:
In terms of the infrastructure, definitely. What I don't know is how well MPI runs through containers, specifically enroot. I believe the [pyxis](https://github.com/NVIDIA/pyxis) module is required, but I remember hearing that some extra change might be required somewhere in enroot or the slurm driver (maybe for the container to be able to run some slurm commands and talk to the slurm daemon even without slurm installed?).